services/web-app/pages/assets/index/index.module.scss

```diff
@@ -236,6 +236,15 @@
   }
 }
 
+.intro-copy {
+  margin-top: spacing.$spacing-11;
+  @include type.type-style('fluid-heading-03', true);
+
+  &:last-child {
+    margin-bottom: spacing.$spacing-10;
+  }
+}
+
```

Comment:
is this being user? can't find it 😅

Reply:
Yea the roadmap page, but now that you point that out it probably makes sense to pull the styles from here into their own file.

Reply:
@francinelucca updated